client/orderbook/orderbook.go

```diff
@@ -461,6 +433,7 @@ func (ob *OrderBook) Unbook(note *msgjson.UnbookOrderNote) error {
 }
 
 // BestNOrders returns the best n orders from the provided side.
+// NOTE: This is UNUSED, and test coverage is a near dup of bookside_test.go.
```

Comment:
There's a huge chunk of test code in orderbook_test.go for this unused method, and it's is covered just as well by a similar test in bookside_test.go.  Maybe we eliminate it unless we're providing this `OrderBook` instance to Go consumers?

Reply:
I left this for now.